src/main/java/com/borchowiec/coronavirustrackerforpoland/payload/GraphDataResponse.java

```diff
@@ -32,20 +32,20 @@ public void setDate(LocalDate date) {
     public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
-        AllConfirmedResponse that = (AllConfirmedResponse) o;
-        return confirmed == that.confirmed &&
+        GraphDataResponse that = (GraphDataResponse) o;
```

Comment:
Why you making new variable? It seems like you only changed the name of argument (from "o" to "that")

Reply:
I understand now, thank u

Reply:
Understand it now! Thank u